src/html_section.js

```diff
@@ -34,6 +34,19 @@ var HTMLSectionBuilder = function(){
 	this.stack = [new HTMLSection()];
 };
 
+HTMLSectionBuilder.scopify = function(renderer) {
```

Comment:
Is this a blocker? The other functions aren't quite what we need to do and this one handles it well.